CordovaLib/CordovaLib/Classes/CDVBridge.m

```diff
@@ -74,7 +74,8 @@ - (void) exec:(NSString*) callbackId withService:(NSString*) service andAction:(
     SEL normalSelector = NSSelectorFromString(methodName);
     if ([obj respondsToSelector:normalSelector]) {
         // [obj performSelector:normalSelector withObject:command];
-        objc_msgSend(obj, normalSelector, command);
+        void (*msgSend)(id, SEL, CDVInvokedUrlCommand*) = (void (*)(id, SEL, CDVInvokedUrlCommand*)) objc_msgSend;
```

Comment:
For me, this has the same issues

```
platforms/osx/CordovaLib/CordovaLib/Classes/CDVBridge.m:78:22: error: too many arguments to function
      call, expected 0, have 3
        objc_msgSend(obj, normalSelector, command);
        ~~~~~~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
In module 'ObjectiveC' imported from /Users/istoica/Workspace/unishell/platforms/osx/CordovaLib/CordovaLib/Classes/CDVBridge.m:22:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk/usr/include/objc/message.h:63:1: note:
      'objc_msgSend' declared here
objc_msgSend(void /* id self, SEL op, ... */ )
^
1 error generated.

warning: ONLY_ACTIVE_ARCH=YES requested with multiple ARCHS and no run destination to provide an active architecture; building for all applicable architectures (in target 'HelloCordova' from project 'HelloCordova')
** BUILD FAILED **
```

What worked was patching it like this

```
        // objc_msgSend(obj, normalSelector, command);
        ((void (*)(id, SEL, id))objc_msgSend)(obj, normalSelector, command);
```

Reply:
+1 for casting at the call site. It seem a bit more obvious what's going on.

By the way, the cast is the right fix, according to https://www.mikeash.com/pyblog/objc_msgsends-new-prototype.html